blocks/social/social.css

```diff
@@ -222,4 +224,9 @@ main .section.social-container a.category-title {
   .block.social {
     right: -10rem;
   }
+
+  main .section.narrow.social-container > .section-container {
```

Comment:
I think we need a combination of `margin-right` and `margin-left` because at certain resolutions the page shifts to the left
<img width="1361" alt="Screenshot 2023-11-13 at 10 40 26" src="https://github.com/hlxsites/sunstar/assets/44490413/b93a222f-371e-4a71-8878-564e3deb620c">

Reply:
> I think we need a combination of `margin-right` and `margin-left` because at certain resolutions the page shifts to the left 

Thanks for pointing this out. This should be addressed now. Please check.

Reply:
```
margin-left: auto;
margin-right: auto;
```
seems to do the trick (the page ever so slightly shifts to the right, but it seems acceptable)